resource/ckb.toml

```diff
@@ -78,7 +78,7 @@ max_cycles = 200_000_000_000
 max_verfify_cache_size = 100_000
 
 [script]
-runner = "Assembly" # {{
+runner = "Rust" # {{
```

Comment:
Won't this changes the default behavior on Linux/Mac to use Rust based VM?

Reply:
Nope, we require exporting ckb.toml before running the node. The default value in the exported config file will be Assembly.